crypto/bls12381/key.go

```diff
@@ -0,0 +1,111 @@
+//go:build !bls12381 || !((linux && amd64) || (linux && arm64) || (darwin && amd64) || (darwin && arm64) || (windows && amd64))
+
+package bls12381
+
+import (
+	"github.com/cometbft/cometbft/crypto"
+)
+
+const (
+	// PrivKeySize defines the length of the PrivKey byte array.
+	PrivKeySize = 32
+	// PubKeySize defines the length of the PubKey byte array.
+	PubKeySize = 48
+	// SignatureLength defines the byte length of a BLS signature.
+	SignatureLength = 96
+	// KeyType is the string constant for the bls12_381 algorithm.
+	KeyType = "bls12_381"
+	// Enabled indicates if this curve is enabled.
+	Enabled = false
+)
+
+// -------------------------------------.
+const (
+	PubKeyName = "cometbft/PubKeyBLS12_381"
```

Comment:
To be consistent with other `PubKeyName`, it should be `cometbft/PubKeyBls12_381`

Reply:
Also, given the importance of this, it may be worth defining this in a common place (not under conditional build) so that it is defined only once (avoiding the risk of someone later changing it only in one place and it falling out of sync). If you go for this approach, then the consts above could also be moved.

Reply:
Good idea 👍